misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobQueueConfig.kt

```diff
@@ -52,5 +54,61 @@ class AwsSqsJobQueueConfig(
    * We use the default retry strategy with SQS, which retries 3 times.
    * As a result, your app could potentially spend 3 x this timeout talking to SQS.
    */
-  val sqs_sending_request_timeout_ms: Int = 5000
+  val sqs_sending_request_timeout_ms: Int = 5000,
+
+  val aws_sqs_job_receiver_policy: AwsSqsJobReceiverPolicy = AwsSqsJobReceiverPolicy.ONE_FLAG_ONLY
 ) : Config
+
+
+/**
+ * AWS SQS consumers are spun by each of a service's pods. Each pod is responsible for running some
+ * number on consumers and making sure they are within the parameters of the feature-flags used.
+ *
+ * Which flags?
+ *  * [CONSUMERS_PER_QUEUE]: This flag specifies the total number of consumers across the entire
```

Comment:
Maybe it will be too much work to migrate everyone and it's outside of the scope of this PR, but should these feature flags really be config values?

Reply:
I could see it going both ways, though the feature flag has been nice to dynamically make changes without deploy 🤷